e2e/ansible/playbooks/resiliency/test-network-failure-rep/test-nw-failure-rep.yml

```diff
@@ -0,0 +1,214 @@
+# test-network-failure-rep.yml
+# Author: Sudarshan
+# Description: Include resiliency test suite in OpenEBS e2e (Test: periodic replica network failures)
+
+###############################################################################################
+#Test Steps:
+#1. Copy Test artifacts to Kubemaster.
+#2. Deploy Percona application with liveness probe running db queries continuously.
+#3. Deploy Pumba netem daemonset on all the nodes.
+#4. Gather Node,Replica Container and Pumba netem details required for test.
+#5. Induce periodic network failures on OpenEBS repilca container.
+#6. Check percona application status to veriy openebs replica sustain network failures.
+#7. Perform cleanup of test artifacts.
+###############################################################################################
+- hosts: localhost
+ 
+  vars_files: 
+    - test-nw-failure-rep-vars.yml 
+ 
+  tasks:
+   - block:
+
+       ###################################################
+       #                PREPARE FOR TEST                 #
+       # (Place artifacts in kubemaster, start logger &  # 
+       # confirm OpenEBS operator is ready for requests. #
+       ###################################################
+
+       - name: Get $HOME of K8s master for kubernetes user
+         shell: source ~/.profile; echo $HOME
+         args: 
+           executable: /bin/bash
+         register: result_kube_home
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Get percona spec and liveness scripts
+         get_url:
+           url: "{{ item }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         with_items: "{{ percona_links }}"
+
+       - name: Replace volume-claim name with test parameters
+         replace:
+           path: "{{ result_kube_home.stdout }}/percona.yaml"
+           regexp: '{{ item.0 }}'
+           replace: '{{ item.1 }}'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+         with_together:
+           - "{{replace_item}}"
+           - "{{replace_with}}"
+         
+       - name: Start the log aggregator to capture test pod logs
```

Comment:
task fails as stern is not available. Add stern binary and run tests. Also can you attach the stern log output as a text file.

Reply:
Added stern bin, o/p:
```